wasm/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+use css_inline as rust_inline;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub fn inline(
+    html: &str,
+    remove_style_tags: Option<bool>,
+) -> Result<String, JsValue> {
+    let options = rust_inline::InlineOptions {
+        remove_style_tags: remove_style_tags.unwrap_or(false),
+        base_url: None,
+        load_remote_stylesheets: false,
+    };
+    let inliner = rust_inline::CSSInliner::new(options);
+    Ok(inliner.inline(html).unwrap())
```

Comment:
To have a more explicit behavior on errors we can add a newtype wrapper for `rust_inline::InlineError` and implement `From` for `JsValue`:

```rust
struct InlineErrorWrapper(rust_inline::InlineError);

impl From<InlineErrorWrapper> for JsValue {
    fn from(error: InlineErrorWrapper) -> Self {
        JsValue::from_str(error.0.to_string().as_str())
    }
}
```

Then we can use `?` syntax instead of `unwrap`:

```rust
Ok(inliner.inline(html).map_err(InlineErrorWrapper)?)
```

Lets say we have an invalid `@` rule specified in a "style" tag (`css-inline` produces an error in this case):

```html
<html><head><style>h1 {background-color: blue;}</style></head><body><h1 style="@wrong { color: ---}">Hello world!</h1></body></html>
```

Then the error handling change will lead to the actual error message, that happened inside:

```
Uncaught (in promise) Invalid @ rule: wrong
```

Instead of the one below, which happens with `unwrap`:

```
Uncaught (in promise) RuntimeError: unreachable
```